docs/docs/testing.md

```
@@ -8,7 +8,26 @@ sidebar_position: 7
+## Making Your System an SNMP Server
+
+Switchmap uses SNMP to gather data from network devices. As a developer it may be difficult to get access to test equipment. This section outlines how to run SNMP on your local machine as an SNMP server.
+
+1. When the server is running and correctly configured, it will provide data about its interfaces via SNMP.
+1. You'll need to use an SNMP client to extract the data from the server.
+
+Here are some links to get you started:
+
+1. Windows 11:
+    1. SNMP Server: 
+        - Setting up the Server: https://blog.paessler.com/how-to-enable-snmp-on-your-operating-system
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)